src/App.js

```diff
@@ -1,14 +1,14 @@
 import { Route, Routes } from 'react-router-dom';
-import Home from './pages/Home';
-import Categories from './pages/CategoriesPage';
-import Navigation from './components/Header';
+import Books from './pages/Books';
+import Categories from './pages/Categories';
+import Navigation from './components/Navigation';
 
 function App() {
   return (
     <>
       <Navigation />
       <Routes>
-        <Route path="/" element={<Home />} />
+        <Route path="/" element={<Books />} />
         <Route path="/categories" element={<Categories />} />
       </Routes>
     </>
```

Comment:
- [ ] In the `index.js` you need to wrap the entire App with a Provider linked to the store so that all components have access to the store.

  See example:

  ```
  import store from './redux/configureStore';

  ReactDOM.render(
  <Provider store={store}>
      <App />
  </Provider>,
  document.getElementById('root')
  )
  ```